airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/auth/AuthButton.tsx

```diff
@@ -39,8 +40,9 @@ function getAuthenticateMessageId(connectorDefinitionId: string): string {
 }
 
 export const AuthButton: React.FC = () => {
-  const { selectedService, authErrors, selectedConnector } = useServiceForm();
-  const hasAuthError = Object.values(authErrors).includes("form.empty.error");
+  const { selectedService, selectedConnector } = useServiceForm();
+  const { hiddenAuthFieldErrors } = useAuthentication();
+  const requiresAuthError = Object.values(hiddenAuthFieldErrors).includes("form.empty.error");
```

Comment:
should this be `requiredAuthError`?

Reply:
I think we parse the word differently :D I always parsed it as "we have an error, that our form 'requires auth'". I changed it to `authRequiredError` now, that might be less ambigious.

Reply:
Ah!  I parsed it as "this requires an auth error" 😜